content/en/docs/armory-admin/prometheus-monitoring.md

```diff
@@ -54,92 +59,170 @@ Access the Prometheus web interface by using the `kubectl port-forward` command.
 
 Navigate to `http://localhost:9090/targets`.
 
-## Configure monitoring in Spinnaker
-
-To enable monitoring of Spinnaker by Prometheus, enable the `metric-stores` configuration.
+## Grant Prometheus RBAC permissions
 
-**Halyard**
+There are two steps to configure Prometheus to monitor Spinnaker:
 
-Issue these halyard commands from within your hal directory or within your halyard container:
+- Add permissions for Prometheus to talk to the Spinnaker namespace
+- Configure Prometheus to discover the Spinnaker endpoints
 
-```bash
-halyard-0:~ $ hal config metric-stores prometheus enable
+Add permissions for Prometheus by applying the following configuration to your cluster. You can learn more about this process on the
+[Prometheus Operator homepage](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/rbac.md).  
 
-+ Get current deployment
-  Success
-+ Edit prometheus metric store
-  Success
-+ Successfully enabled prometheus
+Example config: 
 
-halyard-0:~ $ hal deploy apply
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+  - kind: ServiceAccount
+    name: prometheus
+    namespace: monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/metrics
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources:
+  - configmaps
+  verbs: ["get"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: monitoring
+  name: prometheus
```

Comment:
Prometheus release-0.4 ServiceAccount is called prometheus-k8s. This makes prometheus fail for permissions. Might want to add a note about this.

Reply:
Actually we dont even require the whole Grant Prometheus RBAC permissions section. By installing the release-0.4 of prometheus-kube the RBAC are there by default. @brian-armory  Should we remove it?